mkdocs.yml

```diff
@@ -0,0 +1,84 @@
+---
+site_name: Ansible <Projectname> documentation
+site_url: https://ansible.readthedocs.io/<projectname>
+repo_url: https://github.com/ansible/<projectname>
+edit_uri: blob/main/docs/
+docs_dir: docs
+strict: true
+use_directory_urls: false
+
+theme:
+  name: "ansible"
+  features:
+    - content.code.copy
+    - content.action.edit
+    - navigation.expand
+    - navigation.sections
+    - navigation.instant
+    - navigation.indexes
+    - navigation.tracking
+    - toc.integrate
+  palette:
+    - media: "(prefers-color-scheme: light)"
+      primary: teal
+      accent: blue
+      scheme: default
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: teal
+      accent: blue
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+  locale: en
+
+nav:
+  - index.md
+
+exclude_docs:
+  README.md
+
+plugins:
```

Comment:
Consider removing `plugins` and `markdown_extensions` so this template is really minimal.

Reply:
what about the toc stuff at the end? that seems important to get the same toc settings at least?

My goal is that a project owner doesn't have to touch these files unless they really do want to do something 'different'. So I'm fine with going minimal, as long as minimal gets us common docsites for example.

Reply:
I'm just cautious of overdocumenting to the point where we're basically adding mkdocs documentation. I think we could get away with dropping everything after line 40 or so (after the `nav`) and then adding a comment to say that project owners should consider adding plugins and extensions for some additional bells and whistles.

seems like if we have the ansible community theme with mkdocs then that's the minimum baseline.